kieserver/image.yaml

```diff
@@ -179,6 +179,9 @@ envs:
     - name: "KIE_SERVER_JMS_AUDIT_TRANSACTED"
       example: "false"
       description: "determines if JMS session is transacted or not - default true."
+    - name: "MYSQL_ENABLED_TLS_PROTOCOLS"
```

Comment:
It seems odd for this to be the only database-related env in this yaml. (And also that it's very specific to MySQL.)
I wonder if it would be better to move it into one of these module.yaml files instead?
https://github.com/jboss-container-images/jboss-kie-modules/blob/master/jboss-kie-db-drivers/module.yaml
https://github.com/jboss-container-images/jboss-kie-modules/blob/master/jboss-kie-kieserver/module.yaml

Reply:
Maybe the jboss-kie-db-drivers one doesn't make sense, since the script processing is in jboss-kie-kieserver. Honestly, I'm fine with leaving it in the image.yaml with the other envs. It just stands out like a sore thumb, that's all.